circuitbreaker/umbrel-app.yml

```diff
@@ -3,16 +3,16 @@ id: circuitbreaker
 category: Finance
 name: Circuit Breaker
 version: "v0.4.1"
-tagline: Circuit Breaker is to Lightning what firewalls are to the internet.
+tagline: Your Lightning Node's Firewall
```

Comment:
This might actually be misleading if users expect an actual firewall to be installed by this?!?

Reply:
Yh, fair point. Can we come up with anything shorter than the previous tagline?

Reply:
What is an acceptable length?
Is this a UI concern?

@joostjager thoughts?

Reply:
This honestly is not a huge deal and we can stick with the tagline before.

Generally we optimise for really short and sweet. So that it takes 1 line in the App Store listing: e.g.
![Screenshot 2023-02-02 at 23 49 18](https://user-images.githubusercontent.com/1112574/216476847-98fe84a3-b6e5-443b-947b-2fc4431f93c0.png)

Reply:
> A firewall for Lightning channels

Reply:
>Your Lightning Node's Firewall

I thought this was not so bad. I have a slight preference for using 'node' rather than 'channels'.